feat-content.html

```diff
@@ -13,4 +13,11 @@ <h4 class="feat-content-title">SARS-COV-2 IWC Galaxy Workflows</h4>
             <p class="feat-card-description">A collection of best-practices Galaxy workflows for read mapping, variant calling and other analyses of Illumina data for SARS-CoV-2 sequencing projects maintained by the Intergalactic Workflow Commission (IWC). IWC maintains high quality galaxy workflows that are reviewed and tested before publication and with every new Galaxy release.</p>
         </a>
     </div>
+    <div class="feat-content-card">
+        <a class="cardClick" target="_blank" rel="noopener noreferrer"
+           href="https://tinyurl.com/yck84ayn">
+            <h4 class="feat-content-title">New to containers and workflow languages?</h4>
+            <p class="feat-card-description">A Human Pangenome researcher discusses the motivation behind using these tools in your research and provides a great introduction to Docker and the Workflow Description Language (WDL).</p>
```

Comment:
Just making sure... `Docker` and not `Dockstore`? I haven't seen the videos, so it's probably Docker, but just double-checking.

Reply:
> I guess we'll see how 3 tiles format in dev.

Yes! That's what I figured. If it warps, I can remove one of the existing cards.

Reply:
> Just making sure... `Docker` and not `Dockstore`? I haven't seen the videos, so it's probably Docker, but just double-checking.

Yes. The title says Introduction to Genomic workflows in HPRC. I hadn't yet watched the entire video but will do so.